setup.py

```diff
@@ -18,9 +18,6 @@
     package_data={'sqlalchemy_redshift': ['redshift-ca-bundle.crt']},
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
     install_requires=[
-        'psycopg2>=2.5',
-         # requires sqlalchemy.sql.base.DialectKWArgs.dialect_options, new in
-         # version 0.9.2
```

Comment:
I might need help to rephrase it, since it the other way around, as it seems for me that sqlalchemy.sql.base.DialectKWArgs.dialect_options requires something from psycopg2 that was introduced in 2.5